data/sqlutil/sql.go

```diff
@@ -22,7 +23,7 @@ import (
 func FrameFromRows(rows *sql.Rows, rowLimit int64, converters ...Converter) (*data.Frame, error) {
 	types, err := rows.ColumnTypes()
 	if err != nil {
-		return nil, err
+		return nil, backend.DownstreamError(err)
```

Comment:
I haven't seen this one erroring. I probably wouldn't change this to downstream. The same for `rows.Columns` as it retruns the same errors.